config/environments/test.rb

```diff
@@ -32,11 +32,14 @@
 
   # Show full error reports and disable caching.
   config.consider_all_requests_local = true
-  config.action_controller.perform_caching = false
-  config.cache_store = :null_store
+  # Render exceptions via ErrorsController
+  config.exceptions_app = routes
```

Comment:
This reflects a configuration that we also have in production, and it's necessary in order for the `spec/features/error_handling_spec.rb` tests to pass.